test/integration/smoketest.test.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const chai = require('chai'),
+  Sequelize = require('sequelize'),
+  expect = chai.expect,
+  sinon = require('sinon'),
+  Support = require('./support'),
+  DataTypes = require('sequelize/lib/data-types'),
+  current = Support.sequelize,
+  Op = Sequelize.Op;
+
+describe(Support.getTestDialectTeaser('Model'), () => {
```

Comment:
Move this test to https://github.com/Mhashh/mhashh-sequelize-fork/blob/v6/test/integration/model.test.js

Reply:
completed